motr/st/utils/motr_pver_assign_st.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+#set -x
+
+motr_st_util_dir=$( cd "$(dirname "$0")" ; pwd -P )
+m0t1fs_dir="$motr_st_util_dir/../../../m0t1fs/linux_kernel/st"
+
+# Re-use as many m0t1fs system scripts as possible
+. "$m0t1fs_dir"/common.sh
+. "$m0t1fs_dir"/m0t1fs_common_inc.sh
+. "$m0t1fs_dir"/m0t1fs_client_inc.sh
+. "$m0t1fs_dir"/m0t1fs_server_inc.sh
+. "$m0t1fs_dir"/m0t1fs_sns_common_inc.sh
+
+. "$motr_st_util_dir"/motr_local_conf.sh
+. "$motr_st_util_dir"/motr_st_inc.sh
+
+N=6
+K=2
+S=0
+P=15
+stride=32
+BLOCKSIZE=""
+BLOCKCOUNT=""
+OBJ_ID1="1048577"
+OBJ_ID2="1048578"
+export MOTR_CLIENT_ONLY=1
```

Comment:
I think this can also be removed.

Reply:
No.  It will handle ha events. In this test, we are failed two devices, so need to handle ha events. Please check locally.

Reply:
okay.

Reply:
@yatin-mahajan 
Test is not proceeding without export MOTR_CLIENT_ONLY=1 
and stuck at 
`lt-m0ham: Connecting to 192.168.55.56@tcp:12345:33:1.
motr[98261]:  9c30   WARN  [ha/entrypoint.c:565:ha_entrypoint_client_fom_tick]  rlk_rc=-110
motr[98261]:  9c30   WARN  [ha/entrypoint.c:565:ha_entrypoint_client_fom_tick]  rlk_rc=-110`

It is needed to handle ha events